src/pages/DashboardPage.tsx

```diff
@@ -36,7 +47,7 @@ function DashboardPage(props: Props) {
                 <Icon name="hand paper" />
                 <Header.Content>
                   <Header.Subheader>Dashboard</Header.Subheader>
-                  {t('dashboard.welcome')}
+                  {joke ? 'Trek een bak' : t('dashboard.welcome')}
```

Comment:
Als het het écht netjes wilt doen, maak je van `joke` een functie. Je kunt dit doen door je code (regel 29 t/m 39) in een functie te stoppen. Je begint dan met `const beerTime = () => {...}` en stopt je code op de plaats van de drie puntjes. In plaats van `joke = true` en `joke = false`, doe je `return true` in je `if`-statement en helemaal aan het einde van je functie `return false`.